app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/DefaultChannelAdapter.kt

```diff
@@ -72,25 +72,27 @@ class DefaultChannelAdapter(
                 channelConfig.onClickTelemetry?.invoke(tile)
             }
 
-            itemView.setOnLongClickListener {
-                channelConfig.onLongClickTelemetry?.invoke(tile)
-                val dialog = Dialog(context, R.style.DialogStyle)
-                dialog.setContentView(R.layout.dialog_channel_tiles)
-                dialog.window?.setDimAmount(0.85f)
-
-                dialog.titleText.text = tile.generateRemoveTileTitleStr(context)
-                dialog.removeTileButton.setOnClickListener {
-                    _removeEvents.onNext(tile)
-                    dialog.dismiss()
+            if (channelConfig.itemsMayBeRemoved) {
```

Comment:
nit: while we're changing these lines, might be nice to put this in a helper function.